mote/projects/base/library/browser/patterns/atoms/picture/variations/example/data.yaml

```diff
@@ -0,0 +1,23 @@
+sources:
+    - srcset:
+        attrs:
```

Comment:
This is one piece of backwards compatibility I am going to have to insist on. I neglected the fact that every `<source>` tag can have attributes, whereas before it was just a list of images.